content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -9,4 +9,22 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-This page is under construction.
+Ambient splits Istio’s functionality into two distinct layers, a secure overlay layer and a Layer 7 processing layer.
+The waypoint proxy is an optional component that is Envoy-based and handles L7 processing for different resources.
+What is unique about the waypoint proxy is that it runs outside of the application pod. A waypoint proxy can install,
+upgrade, and scale independently from the application, as well as reduce operational costs. When deploying a waypoint,
+you can configure the waypoint to process traffic for different resource types such as `service` or `workload` or `all`.
+
+## Do you need a waypoint proxy?
+
+
+## Deploy a waypoint proxy
```

Comment:
Would this section be the one that mentions the `--waypoint-for` annotation?

Reply:
yes! label btw :)

Reply:
Ahh ok, that[ just changed](https://github.com/istio/istio/commit/3a34027b570e513f407c6a4de8af58ed16969e7d)! Didn't realize, thank you (: